test/cloudtest/test_privatelink_connection.py

```diff
@@ -35,3 +35,31 @@ def test_create_privatelink_connection(mz: MaterializeApplication) -> None:
     )[0][0]
 
     exists(resource=f"vpcendpoint/connection-{connection_id}")
+
```

Comment:
You have to run `ALTER SYSTEM SET max_aws_privatelink_connections TO 1` for the above test to pass. Unfortunately it doesn't look like the `sql` API allows you to specify the port so you'll have to either update the API so you can run something on the internal port or find a different way.

Reply:
> so you can run something on the internal port

That's exactly what I was thinking would make sense!